plugins/table.md

```diff
@@ -203,13 +205,45 @@ will force pixel resizing. The default is to automatically detect what the table
 
 ```js
 tinymce.init({
-  selector: "textarea",  // change this value according to your HTML
-  plugins: "table",
-  menubar: "table",
-  toolbar: "table",
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'table',
+  menubar: 'table',
+  toolbar: 'table',
   table_responsive_width: false
 });
 ```
+
+### `table_sizing_mode`
+
+{{site.requires_5_4v}}
+
+The `table_sizing_mode` option enforces the table sizing method used for new and modified tables (including resizing operations on tables). This option only impacts the _width_ of tables and cells and does not apply to the _height_ of tables and cells.
+
+This option accepts the following options:
```

Comment:
"option options"?

Reply:
changed second 'option' to 'value' in https://github.com/tinymce/tinymce-docs/pull/1593/commits/a879dfa62084003a4c224c49609fe9f349bd9d00